docs/SETUP.md

```diff
@@ -70,6 +70,25 @@ We use Postgres around these parts. Installation will differ based on your OS.
   * Other [linux instructions](https://www.postgresql.org/download/) (server and developer libraries needed)
 * Run the command `rails db:create && rails db:migrate` to set up the tables
 
+* Configure Postgres
+ * Setup your user; the suggested password is `postgres` if you only use this machine for development purposes
+   ```
+   sudo -u postgres createuser -s `whoami`
+   ```
+ * Set your env variables:
+   * Mac:
+     ```
+     echo "export POSTGRES_USER=`whoami`" >>~/.bashrc
+     echo "export POSTGRES_PASSWORD=postgres" >>~/.bashrc
+     echo "export PGDATABASE=dcaf_case_management_development" >>~/.bashrc
+     ```
```

Comment:
Feedback requested from a Mac user: do these steps result in the values appearing in your ENV on next terminal start up? If not, which file should they go in?

Reply:
Macs default to zsh now, obnoxiously, which makes it tough to write directions like this. But the tldr is that I think if you are running a recent mac os your default shell rc file is going to be `~/.zshrc` so I think we prrrrrrrob want to point that in that direction.